templates/html-header.twig

```diff
@@ -36,7 +36,7 @@
 	{% if (og_description) %}
 		<meta name="twitter:description" content="{{ og_description|e('html_attr')|raw }}">
 	{% endif %}
-	<meta name="twitter:creator" content="{{ author_override ?? fn( 'get_the_author_meta', 'display_name', post.post_author ) }}">
+	<meta name="twitter:creator" content="{{ post.author.name }}">
```

Comment:
please add a check if post.author is set (not null), for cases that is not set for some reason.
https://github.com/timber/timber/blob/master/lib/Post.php#L944

Reply:
Done.